src/unk_02054884.c

```diff
@@ -161,22 +152,20 @@ Pokemon * sub_02054A74 (const Party * param0)
 
 BOOL Party_HasTwoAliveMons (const Party *party)
 {
-    return sub_02054A10(party) >= 2;
+    return Party_AliveMonsCount(party) >= 2;
 }
 
-void sub_02054AC4 (Party * param0)
+void Party_SetChampionRibbons (Party *party)
```

Comment:
```suggestion
void Party_GiveChampionRibbons (Party *party)
```

My logic here is that `SetChampionRibbons` implies to me that I can specify whether the members have the ribbon or not. `Give` is unambiguous; after this routine, all party members will have the ribbon flag flipped to `TRUE`.